test/models/cart_test.rb

```diff
@@ -0,0 +1,17 @@
+class CartTest < ActiveSupport::TestCase
+  test 'cart has initial contents' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.